### ISSUE TITLE
No Request URL for PRA

### ISSUE THREAD

**Issue by rgmccaw** [oracle/terraform-provider-baremetal#175]
## Description of the bug:

For the `baremetal_objectstorage_preauthrequest` resource there is no Pre-Authorized Request URL in the attributes available for output.

The url is needed in order to make use of the PRA.

## A self-contained terraform file that reproduces the issue:

```
#     ___  ____     _    ____ _     _____
#    / _ \|  _ \   / \  / ___| |   | ____|
#   | | | | |_) | / _ \| |   | |   |  _|
#   | |_| |  _ < / ___ | |___| |___| |___
#    \___/|_| \_/_/   \_\____|_____|_____|

## PAR(preauthenticated requests) management.
## This example creates two preauthenticated requests, one on a bucket and the second on an object
## Export the environmental variables used for configuration.
## Provide the script with an object and bucket present in your namespace

variable "tenancy_ocid" {}
variable "user_ocid" {}
variable "fingerprint" {}
variable "private_key_path" {}
variable "compartment_ocid" {}
variable "namespace_name" {}

variable "bucket_name" {}
variable "object_name" {}

provider "baremetal" {
  region = "us-phoenix-1"
  alias = "phx"
  tenancy_ocid = "${var.tenancy_ocid}"
  user_ocid = "${var.user_ocid}"
  fingerprint = "${var.fingerprint}"
  private_key_path = "${var.private_key_path}"
}

resource "baremetal_objectstorage_preauthrequest" "parOnObject" {
  namespace = "${var.namespace_name}"
  bucket = "${var.bucket_name}"
  object = "${var.object_name}"
  name = "parOnObject"
  access_type = "ObjectRead" //Other configurations accepted are ObjectWrite, ObjectReadWrite
  time_expires = "2019-11-10T23:00:00Z"
}

output "pra_request_url" {
  value = "${baremetal_objectstorage_preauthrequest.parOnObject.access_uri}" # access_uri or similar does not exist
}
```

**Comment by eginez** [comment]
Hey there thanks for the bug, I am little confuse.
Is it that the url is `acces_uri` is empty? or is it that the access_uri complete

**Comment by eginez** [comment]
Never mind I see what you mean that schema is missing the "access_uri" field

**Comment by codycushing** [comment]
@eginez Im not seeing the `access_uri` defined in `preauthenticatedRequestSchema`, it would be a computed value.

**Comment by rgmccaw** [comment]
Issue is fixed in release v1.0.15

**Comment by rgmccaw** [comment]
@eginez @codycushing thank you for the quick response.
